app/code/Magento/InventoryInStorePickupApi/Api/GetPickupLocationInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
```

Comment:
Please remove declaring strict types from php interfaces
see 1.3.1 https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html#1-basic-programming-principles

Reply:
Good point to discuss.
@ishakhsuvarov why we generally not follow this recommendation?

Reply:
@swnsma It became a standard in MSI repository before 1.3.1 was introduced. I guess this is a power of habit at this point. Personally I don't see any harm in it, but of course we would like to follow guidelines for new code.